### DISCUSSION TITLE
Tab - How to navigate around disabled tab?

### DISCUSSION THREAD

**Discussion by binhxn** [ariakit/ariakit#1546] [Questions] [answered]
Hello!

I was wondering how to navigate around a disabled Tab? For example, let's say I have 3 tabs and Tab2 is disabled, I'd like to navigate from Tab1 -> Tab3 when pressing the Right arrow key _once_. I can do this in Reakit using the `focusable` prop on Tab, but that seems to be removed in Ariakit. I was about to reach for `selectOnMove` state, but realized that's not the right solution.

Here's a disabled Tab [example](https://codesandbox.io/s/reakit-disabled-tab-shc5gz?file=/src/App.tsx) using Reakit.

Any ideas on how to approach this?

**Comment by diegohaz** [comment, marked as answer]
The prop has been renamed to accessibleWhenDisabled.

**Reply by binhxn** [reply]
Awesome, that's what I was looking for. Thank you!
